driver/collector/postgres_collector.py

```diff
@@ -361,11 +363,12 @@ def __init__(
         directly and instead use the collector_factory.get_collector method instead.
 
         Args:
-            conn: The connection to the database
+            conns: The connections to the database (one for each logical database)
             options: Options used to define which tables to use for metric collection
         """
-        self._conn = conn
+        self._conns = conns
         self._version_str = version
+        self._main_logical_db = main_logical_db
```

Comment:
how does the user determine the `main` logical DB out of curiosity? The user will provide a list of logical DBs right? Curious how we determine which one is main.

Reply:
It's the POSTGRES_OTTERTUNE_DB_NAME, using that to preserve backwards compatibility

Reply:
Do we want to keep that as the variable for the main db? I know the name isn't exactly right, but it should make migration to the new client easier.